chains/solana/contracts/programs/ccip-router/src/instructions/v1/token_admin_registry.rs

```diff
@@ -53,20 +60,91 @@ pub fn register_token_admin_registry_via_owner(
 }
 
 pub fn set_pool(
-    ctx: Context<ModifyTokenAdminRegistry>,
+    ctx: Context<SetPoolTokenAdminRegistry>,
```

Comment:
I'm wondering how this behaves when we'll have to deal with more than one Token Pool per token